### ISSUE TITLE
Why using SVM to train in the   facenet/src/classifier.py training process

### ISSUE THREAD

**Issue by yuanzhenjie** [davidsandberg/facenet#438]
你好，git 上的davidsandberg/facenet 是对facenet 完全的实现么
他的acenet/src/classifier.py  train   model.fit(emb_array, labels) （＃96）中 训练 是直接使用SVM  ,这与和 facenet貌似没什么关系

Hello, the training of  acenet/src/classifier.py "model.fit (emb_array, labels) (#96)"  used SVM, which seemed to have nothing to do with facenet。

**Comment by mingfengwuye** [comment]
I think it is only a demo, you can use KNN or other classifier. Facenet is used to extract features.

**Comment by yuanzhenjie** [comment]
thanks, 
I know that  the facenet  is used to  extract features. And I think that  the facenet need to train ,to make  the supper parameters file of the  facenet.  
how to train  the facenet not training classifier? 
In the **Face Recognition using Tensorflow**
 [https://github.com/davidsandberg/facenet/wiki](url) ,the train process is more like a process of  training classify with inception resent v1 [https://github.com/davidsandberg/facenet/wiki/Classifier-training-of-inception-resnet-v1](url) .(Maybe I am wrong or I miss something).

**Comment by yuanzhenjie** [comment]
[ddffd](https://github.com/davidsandberg/facenet/wiki)

**Comment by GGLW123** [comment]
if facenet is used to extract features, is there a proper way to do prediction with softmax layer?

**Comment by lizhili** [comment]
@yuanzhenjie 
你好，这点我也很困惑。在看对facenet的简介中，介绍了其对800万人的两亿张图片进行分类，，但是我看大家都在说，facenet是特征提取，还要加个knn分类。。。
如果要做成1：N的识别库，自己还要再训练一个分类器的话，感觉后面这一部分算法很影响性能

**Comment by xxllp** [comment]
这个我也不是很理解，不过现在有更多新的算法了

**Comment by justein** [comment]
> 这个我也不是很理解，不过现在有更多新的算法了

你们用的什么算法？
